features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesNode.kt

```diff
@@ -230,6 +241,10 @@ class MessagesNode @AssistedInject constructor(
         callbacks.forEach { it.onViewKnockRequests() }
     }
 
+    private fun Context.sameRoomToast() {
+        context.toast("Already viewing this room!")
```

Comment:
We could remove the `Context` receiver here. Also maybe a good opportunity to add this string in Localazy?